packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -297,15 +299,30 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
       const videoOptions = { localVideoStreams: this.localStream ? [this.localStream] : undefined };
 
       const isTeamsMeeting = !('groupId' in this.locator);
-      const call = isTeamsMeeting
-        ? this.callAgent.join(this.locator as TeamsMeetingLinkLocator, {
-            audioOptions,
-            videoOptions
-          })
-        : this.callAgent.join(this.locator as GroupCallLocator, {
-            audioOptions,
-            videoOptions
-          });
+      /* @conditional-compile-remove(rooms) */
+      const isRoomsCall = !('roomId' in this.locator);
+
+      let call;
+
+      if (isTeamsMeeting) {
+        call = this.callAgent.join(this.locator as TeamsMeetingLinkLocator, {
+          audioOptions,
+          videoOptions
+        });
+      } else {
```

Comment:
This else will be running even it is Rooms locator, the behavior might be right (cause in the run time callAgent.join points to the same js function), but let's fix this in case that it creates hidden regressions